pinot-tools/pom.xml

```diff
@@ -123,6 +118,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
It seems that `pinot-orc`was included in the original code. Let's not modify this part and revert back to the original copy.